astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -543,16 +542,12 @@ def test_earth_orientation_table():
     altaz = AltAz(location=location, obstime=t)
     sc = SkyCoord(1*u.deg, 2*u.deg)
     # Default: uses IERS_Auto, which will give a prediction.
-    with catch_warnings() as w:
-        altaz_auto = sc.transform_to(altaz)
-
-    assert len(w) == 0
+    altaz_auto = sc.transform_to(altaz)
```

Comment:
Note: Now that we turn all warnings into errors, except for the ones we explicitly ignore, check for zero warning is no longer necessary.